internal/internal_task_handlers_test.go

```diff
@@ -2153,7 +2153,7 @@ func Test_NonDeterministicCheck(t *testing.T) {
 		delete(commandTypes, cmd)
 	}
 
-	require.Equal(t, 15, len(commandTypes), "If you see this error, you are adding new command type. "+
+	require.Equal(t, 17, len(commandTypes), "If you see this error, you are adding new command type. "+
```

Comment:
Note, we expect this will be built by @bergundy and team as they build out Nexus support, this is just needed to pass CI